prime-router/src/testIntegration/resources/datatests/translation-test-config.csv

```diff
@@ -22,8 +22,9 @@ HL7_to_FHIR_to_HL7/elims_29_5065302_35227_NoPII_CANCELED.hl7,,HL7_to_FHIR_to_HL7
 HL7_to_FHIR_to_HL7/hci.hl7,,HL7_to_FHIR_to_HL7/hci-expected.hl7,metadata/hl7_mapping/ORU_R01/ORU_R01-base,HL7,PASS,"","","","",""
 FHIR_to_HL7/sample_AOE_1_20230220-0001.fhir,,FHIR_to_HL7/sample_AOE_1_20230220-0001-custom-datetime.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/default-sender-transform","development.DEV_FULL_ELR",""
 FHIR_to_HL7/sample_AOE_1_20230220-0001.fhir,,FHIR_to_HL7/sample_AOE_1_20230220-0001.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/default-sender-transform","",""
-FHIR_to_HL7/sample_AOE_2_20230220-0001.fhir,,FHIR_to_HL7/sample_AOE_2_20230220-0001.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","",""
+FHIR_to_HL7/sample_AOE_2_20230220-0001.fhir,,FHIR_to_HL7/sample_AOE_2_20230220-0001.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/default-sender-transform","",""
 FHIR_to_HL7/sample_SR_1_20230302-0001.fhir,,FHIR_to_HL7/sample_SR_1_20230302-0001.hl7,metadata/hl7_mapping/receivers/STLTs/CA/CA-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","",""
+FHIR_to_HL7/sample_SR_2_20231024-0002.fhir,,FHIR_to_HL7/sample_SR_2_20231024-0002.hl7,metadata/hl7_mapping/receivers/STLTs/NY/NY-receiver-transform,HL7,PASS,"","","metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform","",""
```

Comment:
Where is the SR_2 hl7? Wondering if it was missing from your commit

Reply:
It is missing! Just added it, thanks!